### PR TITLE
Fix filter_backends inside search action

### PR DESCRIPTION
**Pull request by duda-prc**
By incorporating the API's DEFAULT_FILTER_BACKENDS into `filter_backends` [search action], we refrain from altering the inherent behavior of the API's.

### CODE REVIEW

File: test_app/tests/tests_views/tests_filter_views.py [hunk 1]
```diff
@@ -50,3 +50,18 @@ def test_search_body_malformed(self):
...
+    def test_search_body_ordered(self):
+        teachers = []
+        for i in range(10, 20):
+            teachers.append(TeacherFactory(id=i, is_ghost=False, picture=None))
+        expected_teachers = [self.expected_teacher(teacher) for teacher in teachers]
+
+        for i in range(200, 210):
+            TeacherFactory(id=i, is_ghost=True, picture=None)  # noise
+
+        url = f"{self.url}/search?sort=id"
```
Comment (lucasgomide) [inline test_app/tests/tests_views/tests_filter_views.py:63]: Should the `sort` be provided into querystring?

File: test_app/tests/tests_views/tests_filter_views.py [hunk 2]
```diff
@@ -50,3 +50,18 @@ def test_search_body_malformed(self):
         search = "potato"
         response = self.client.post(url, data=search, content_type="application/text")
         self.assertEqual(status.HTTP_415_UNSUPPORTED_MEDIA_TYPE, response.status_code)
+
+    def test_search_body_ordered(self):
+        teachers = []
+        for i in range(10, 20):
+            teachers.append(TeacherFactory(id=i, is_ghost=False, picture=None))
```
Comment (lucasgomide) [inline test_app/tests/tests_views/tests_filter_views.py:57]: The `id` could be omitted. It seems irrelevant for your test

Comment (duda-prc) [reply test_app/tests/tests_views/tests_filter_views.py:57]: I am sorting by `id` in the test, so that's why I'm specifically including its value.

File: test_app/tests/tests_views/tests_filter_views.py (hunk 1 above)
Comment (duda-prc) [reply test_app/tests/tests_views/tests_filter_views.py:63]: It does not work :(

Screenshot 2024-02-20 at 10 40 23
-----
This is because the `OrderingFilter` inside `rest_framework` is implemented expecting a `query_param`:

Screenshot 2024-02-20 at 10 43 13

**Review by lucasgomide — APPROVED** [review]

**Review by DWalterJansen — APPROVED** [review]
